README.md

```diff
@@ -44,6 +44,15 @@ You can change the script configuration at the beginning of the file:
 
 \*Check the [Useful icons](#useful-icons) section for examples.
 
+Alternatively, these can be set via command-line arguments, such as:
+```ini
+[module/pulseaudio-control]
+type = custom/script
+exec = "polybar-pulseaudio-control --vol-max=150 --vol-icons='🔈 ,🔉 ,🔊 ' --sink-name-from-prop=device.description --osd"
```

Comment:
This doesn't need to be quoted, but GitHub renders it weirdly if it isn't: `--vol-max`, `--vol-icons`, and `--sink-name-from-prop` get highlighted, so it's _almost_ an accidental feature, but then `--osd` doesn't.